src/components/Filters/Filters-container.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import Grid from '@material-ui/core/Grid/Grid';
+
+import useFiltersStyles from './Filters-styles';
+
+const Filters = () => {
+  const classes = useFiltersStyles();
+  const blockchains: string[] = ['BTC', 'ETH', 'XRP', 'LTC', 'BCH', 'XTZ', 'ADA', 'EOS', 'XLM'];
+  const activeClasses = [classes.button, classes.active]
+
+  const filterHandler = (eventTarget: HTMLBodyElement) => {
+    const active: boolean = eventTarget.className === classes.button ? true : false;
+    active && (eventTarget.className = activeClasses.join(' '));
+    !active && (eventTarget.className = classes.button);
+  }
+
+  const renderButtons = (data: string[]) =>
+    data.map((button: string, index: number) => (
+      <button
+        className={classes.button}
+        key={index}
+        onClick={(e: any) => filterHandler(e.target)}
+      >
+        {button}
+      </button>
+    ))
+
+  return (
+    <Grid container justify="flex-start" alignItems="center" className="Container">
```

Comment:
Nie jestem jeszcze do końca pewna, jak działają te style w materialu. Czy ten zapis odpowiada "klasycznemu" stylowi inlinowemu? Da radę to potem jakoś nadpisać w razie W?

Reply:
Tak, ściągnięte z innych Gridów w apce :)

Reply:
OK!
W sumie, jak to się wszędzie powtarza, to można kiedyś przy okazji dorzucić do jakichś bardziej uniwersalnych stylów gridowych. Ale tutaj już raczej głośno myślę ;)

Reply:
Nie wszędzie w TransactionsList nie ma tego

Reply:
`className="Container"` - Why is this neccesary? We are not using css anywhere.